openlibrary/templates/books/custom_carousel.html

```diff
@@ -43,10 +46,19 @@
   <div class="book carousel__item">
     <div class="book-cover">
       <a href="$(url)">
-        <img class="bookcover"
-          width="130" height="200"
-          title="$('%s%s'%(title,byline))"
-          $img_attr="$(cover_url)"/>
+        $if cover_url:
+          <img class="bookcover"
+            width="130" height="200"
+            title="$('%s%s'%(title,byline))"
+            $img_attr="$(cover_url)"/>
+        $else:
+          <div class="carousel__item__blankcover">
+              <div>
+                  <div>$:macros.TruncateString(title, 70)
```

Comment:
The nesting here is strange; let's make these adjacent.

```html
<div class="carousel__item__blankcover">
    <div class="title">Harry Potter</div>
    <div class="byline">J. K. Rowling</div>
</div>
```

Reply:
Can you `suggest` a change for this one?